src/git.zsh

```diff
@@ -142,6 +142,114 @@ _fzf_complete_git() {
         return
     fi
 
+    if [[ $subcommand = 'pull' ]]; then
+        local prefix_option completing_option
+
+        local git_options_argument_required=(--cleanup --date --depth --depthn --negotiation-tip -o -s --server-option --shallow-exclude --shallow-since --strategy --strategy-option --strategy-option=diff-algorithm --upload-pack -X)
```

Comment:
```suggestion
        local git_options_argument_required=(--cleanup --date --depth --deepen --negotiation-tip -o -s --server-option --shallow-exclude --shallow-since --strategy --strategy-option --strategy-option=diff-algorithm --upload-pack -X)
```